### ISSUE TITLE
Having trouble using, any updated examples?

### ISSUE THREAD

**Issue by Kencol3** [davidrpugh/pyBEA#10]
Hi, I'm trying to use this to source data from the BEA. I'm fairly new at this, but it would be really helpful if you could post some examples on how to use pybea and call each series. I think its because a bunch of things have changed. Any help would be much appreciated. Examples are super helpful and this is a blessing what you've done here.

**Comment by davidrpugh** [comment]
@Kencol3 You are correct that much has changed and I need to update the package.  I will try to release a new version with working example notebooks soonest.

**Comment by davidrpugh** [comment]
@Kencol3 I have done a quick fix of the library and example notebooks. A few examples of how to use the library can be found in the [user guide](https://github.com/davidrpugh/pyBEA/blob/master/examples/user-guide.ipynb).  I will add more notebooks to the examples directory as I add more datasets to the API. Let me know if you have any more questions.
